src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/FeederJmapAllSimulation.scala

```diff
@@ -3,7 +3,7 @@ package org.apache.james.gatling.simulation.jmap
 import io.gatling.core.Predef._
 import io.gatling.core.scenario.Simulation
 import org.apache.james.gatling.control.{Password, RandomUserPicker, User, UserFeeder, Username}
-import org.apache.james.gatling.jmap.scenari.FeederJmapAllScenario
+import org.apache.james.gatling.jmap.scenari.JmapAllScenario
 import org.apache.james.gatling.simulation.{Configuration, HttpSettings}
 
 class FeederJmapAllSimulation extends Simulation {
```

Comment:
there is already a simulation for this scenario JmapAllSimulation, you can drop the simulation too